coderd/telemetry/telemetry.go

```diff
@@ -1149,6 +1151,9 @@ type User struct {
 	RBACRoles       []string            `json:"rbac_roles"`
 	Status          database.UserStatus `json:"status"`
 	GithubComUserID int64               `json:"github_com_user_id"`
+	// LoginType is nullable for backwards compatibility.
+	// We used to not set it at all.
+	LoginType *string `json:"login_type"`
```

Comment:
Out of interest, can you not use `omitempty` with a regular `string` type?

Reply:
That would work too, I don’t have a strong preference either way.

Reply:
I feel like that approach is more idiomatic, TBH

Reply:
updated to use omitempty